app/views/users/new.html.erb

```diff
@@ -0,0 +1,39 @@
+</h1>Create a new user </>
+<% 
+=begin %>
+ <form action="/users" method="post" accept-charset="UTF-8">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+</form> 
+<% 
+=end %>
+
+
+ <%= form_tag('/users', method: :post) do %> 
+   <%= label_tag(:username, "Username:") %> 
+   <%= text_field_tag(:username) %> 
+   <%= label_tag(:email, "Email:") %> 
+   <%= email_field_tag(:email) %> 
+   <%= label_tag(:password, "Password:") %> 
+   <%= password_field_tag(:password) %>    
+   <%= submit_tag("Create") %> 
+  <% end %>
+
+
+  
+<% 
+=begin %>
+  <%= form_for (User.new) do |f| %>
```

Comment:
- [ ] You should be using `@user` from the controller here.
![Screen Shot 2020-11-18 at 6 33 34 PM](https://user-images.githubusercontent.com/48804258/99565947-91bbc000-29cc-11eb-95f6-8aa528fd5c25.png)